statistics/selectivity.go

```diff
@@ -345,8 +378,8 @@ func getUsableSetsByGreedy(nodes []*StatsNode) (newBlocks []*StatsNode) {
 	return
 }
 
-// popCount is the digit sum of the binary representation of the number x.
-func popCount(x int64) int {
+// bitCount is the number of bit `1` in the binary representation of number x.
+func bitCount(x int64) int {
```

Comment:
How about use `bits.OnesCount64` instead of it.

Reply:
Yeah, this function is introduced before golang implement the `bit` package. We may replace it now.